### PR TITLE
visible to frontend js works

### PR DESCRIPTION
**Pull request by npwhite**

### CODE REVIEW

File: posts/static/makepost/posts.js [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+    privateDiv.appendChild(removeButtonDiv);
 
     div.appendChild(privateDiv);
 }
 
+function deleteParent() {
+    this.parentNode.parentNode.remove()
+}
+
 // not working, cant handle redirects
```
Comment (FordMcLeod) [inline posts/static/makepost/posts.js:34]: Is this comment just left over? can we delete it or does it not handle redirects? I'm confused what it's saying

**Review by FordMcLeod — APPROVED** [review]

**Comment by FordMcLeod** [comment]
Resolves #73

File: posts/static/makepost/posts.js (hunk 1 above)
Comment (MichaelParadis) [reply posts/static/makepost/posts.js:34]: Yeah this should be removed

Comment (npwhite) [reply posts/static/makepost/posts.js:34]: yeah idk, it was there when I arrived, so it remained after I left lol

**Pull request merged**